frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -187,9 +189,17 @@ export const ClusterVersionDetailsTable: React.SFC<ClusterVersionDetailsTablePro
         </div>
       </div>
       <div className="co-m-pane__body-group">
+        <p className="co-m-pane__explanation">
+          View this cluster and manage subscription settings at
+          {' '}
+          <ExternalLink
+            text="cloud.redhat.com"
+            href={`https://cloud.redhat.com/openshift/details/${clusterID}`}
```

Comment:
@luciddreamz is there a way we can link directly to the cluster details? it looks like this link with the cluster ID doesn't work

Reply:
it doesn't work as of today, but I will make it work :smile: